xsensTFBridge/src/xsensTFBridge.cpp

```diff
@@ -0,0 +1,257 @@
+#include <iostream>
```

Comment:
Order a bit the includes.

As last add the stdc++ libraries.
Maybe you should also remove some headers which I think are not needed